### PR TITLE
Move to sdk 0.6.5

### PR DESCRIPTION
**Pull request by mikevoronov**

### CODE REVIEW

File: air-interpreter/Cargo.toml [hunk 1]
```diff
@@ -26,4 +26,4 @@ serde = { version = "=1.0.118", features = [ "derive", "rc" ] }
 serde_json = "1.0"
```
Comment (boneyard93501) [inline air-interpreter/Cargo.toml:26]: 1.0.56 ?

File: air-interpreter/Cargo.toml [hunk 2]
```diff
@@ -11,12 +11,12 @@ path = "src/wasm_bindgen.rs"
...
-path = "src/fce.rs"
+path = "src/marine.rs"
 
 [dependencies]
 air = { path = "../air" }
 
-fluence = { version = "0.6.2", features = ["logger"] }
+fluence = { version = "0.6.5", features = ["logger"] }
 
 # Keep 0.2.65 until this is resolved https://github.com/rustwasm/wasm-pack/issues/886
```
Comment (boneyard93501) [inline air-interpreter/Cargo.toml:21]: isn't https://github.com/rustwasm/wasm-pack/issues/886 resolved ?

Comment (mikevoronov) [reply air-interpreter/Cargo.toml:21]: It is, but it has to be checked and I think it deserves a separate PR.
@coder11 could you please a look?

File: air-interpreter/Cargo.toml (hunk 1 above)
Comment (mikevoronov) [reply air-interpreter/Cargo.toml:26]: We could fix it, yep, but is it worth so?

Comment (boneyard93501) [reply air-interpreter/Cargo.toml:26]: probably not

File: air-interpreter/Cargo.toml (hunk 2 above)
Comment (boneyard93501) [reply air-interpreter/Cargo.toml:21]: yep

File: air-interpreter/Cargo.toml (hunk 1 above)
Comment (folex) [reply air-interpreter/Cargo.toml:26]: It uses 1.0.61 judging by Cargo.lock

**Review by folex — APPROVED** [review]

**Pull request merged**